aptos-move/framework/move-stdlib/sources/fixed_point32.move

```diff
@@ -268,26 +282,26 @@ module std::fixed_point32 {
             ceil(num)
         }
     }
+
     spec round {
         pragma verify_duration_estimate = 120;
         pragma opaque;
         aborts_if false;
         ensures result == spec_round(num);
     }
+
     spec fun spec_round(val: FixedPoint32): u64 {
         let fractional = val.value % (1 << 32);
         let boundary = (1 << 32) / 2;
         let one = 1 << 32;
-        if (fractional < boundary) {
-            (val.value - fractional) >> 32
-        } else {
-            (val.value - fractional + one) >> 32
+        if (fractional < boundary) {(val.value - fractional) >> 32} else {(val.value - fractional + one) >> 32
         }
     }
 
     // **************** SPECIFICATIONS ****************
 
-    spec module {} // switch documentation context to module level
+    spec module {
```

Comment:
This looks strange to me.

Reply:
we will fix it

Reply:
fixed at https://github.com/movebit/movefmt/commit/a15b8f3b2b461a42e5f0911034bd697ef0d62439